# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../bmon/gcc-4.0.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -ruN bmon-2.1.0-orig/src/out_audio.c bmon-2.1.0/src/out_audio.c
--- bmon-2.1.0-orig/src/out_audio.c	2005-04-05 17:01:33.000000000 +0200
+++ bmon-2.1.0/src/out_audio.c	2010-08-30 10:00:57.456003585 +0200
@@ -141,7 +141,7 @@
 	.om_draw = audio_draw,
 	.om_set_opts = audio_set_opts,
 	.om_probe = audio_probe,
-	.om_shutdown audio_shutdown,
+	.om_shutdown = audio_shutdown
 };
 
 static void __init audio_init(void)
diff -ruN bmon-2.1.0-orig/src/out_db.c bmon-2.1.0/src/out_db.c
--- bmon-2.1.0-orig/src/out_db.c	2005-04-05 17:01:33.000000000 +0200
+++ bmon-2.1.0/src/out_db.c	2010-08-30 10:00:57.456003585 +0200
@@ -679,7 +679,7 @@
 	.om_draw = db_draw,
 	.om_set_opts = db_set_opts,
 	.om_probe = db_probe,
-	.om_shutdown db_shutdown,
+	.om_shutdown = db_shutdown,
 };
 
 static void __init db_init(void)
diff -ruN bmon-2.1.0-orig/src/out_xml_event.c bmon-2.1.0/src/out_xml_event.c
--- bmon-2.1.0-orig/src/out_xml_event.c	2005-04-05 17:01:33.000000000 +0200
+++ bmon-2.1.0/src/out_xml_event.c	2010-08-30 10:00:57.456003585 +0200
@@ -127,7 +127,7 @@
 	.om_draw = xml_event_draw,
 	.om_set_opts = xml_event_set_opts,
 	.om_probe = xml_event_probe,
-	.om_shutdown xml_event_shutdown,
+	.om_shutdown = xml_event_shutdown
 };
 
 static void __init xml_event_init(void)
